app/components/BeelieversAuction/AuctionTable.tsx

```diff
@@ -0,0 +1,98 @@
+import { Card, CardContent } from "../ui/card";
+import { Table } from "../ui/table";
+import { CellProps, Column } from "react-table";
+import { SUIIcon } from "../icons";
+import { trimAddress } from "../Wallet/walletHelper";
+
+interface Bid {
+	rank: number;
+	bidder: string;
+	time: number;
+	amount: string;
+}
+
+export const columns: Column<Bid>[] = [
+	{
+		Header: "Rank",
+		accessor: "rank",
+		Cell: ({ value }: CellProps<Bid>) => <div className="flex items-center gap-2">#{value}</div>,
+	},
+	{
+		Header: "Bidder",
+		accessor: "bidder",
+		Cell: ({ row }: CellProps<Bid>) => (
+			<div className="flex space-x-2">
+				<span>{trimAddress(row.original.bidder)}</span>
+			</div>
+		),
+	},
+	{
+		Header: "Bid Time",
+		accessor: "time",
+		Cell: ({ row }: CellProps<Bid>) => (
+			<div className="flex space-x-2">
+				<span>{row.original.time} hr ago</span>
+			</div>
+		),
+	},
+	{
+		Header: "Bid Amount",
+		accessor: "amount",
+		Cell: ({ row }: CellProps<Bid>) => (
+			<div className="flex space-x-2">
+				<SUIIcon prefix="" className="h-5 w-5" />
+				<span>{row.original.amount} SUI</span>
+			</div>
+		),
+	},
+];
+
+const mockLeaderBoard: Bid[] = [
```

Comment:
You should use data from the server , so move the mock there

Reply:
Done